config/locales/en.yml

```diff
@@ -1133,7 +1133,7 @@ en:
       this_is_private_community: "You need to sign up before you can view the content."
     invitation_form:
       email: "Email address(es)"
-      email_with_limit: "Email addresses (max. %{invitation_limit})"
+      email_with_limit: "Email addresses (max. %{invitation_limit} per day)"
```

Comment:
True, but do you think that user's expects that someone else's invitations affect to their daily amount?